.github/workflows/lint-markdown.yml

```diff
@@ -14,9 +14,11 @@ jobs:
     steps:
       - name: Checkout repo
         uses: actions/checkout@v5
-      - name: Run markdownlint
-        uses: articulate/actions-markdownlint@v1
+      - name: Run markdownlint-cli2
+        uses: DavidAnson/markdownlint-cli2-action@v21
```

Comment:
We suddenly started getting markdownlint errors, and in debugging, I realized that our markdownlint action was deprecated in favor of this one. David Anson also authors the VS Code extension that we recommend in the repo configuration, so this makes CI consistent with VS Code errors, in theory.